gwctl/pkg/printer/policies_test.go

```diff
@@ -354,3 +354,155 @@ timeoutpolicies.bar.com      Direct       Namespaced  5m
 		t.Errorf("Unexpected diff\ngot=\n%v\nwant=\n%v\ndiff (-want +got)=\n%v", got, want, diff)
 	}
 }
+
+func TestPolicyCrd_PrintDescribeView(t *testing.T) {
+	// fakeClock := testingclock.NewFakeClock(time.Now())
```

Comment:
Can we clean this up if we don't need a fakeClock? Else, can we evaluate if we need a fakeClock and use it?